installer/manifests/keptn/charts/continuous-delivery/templates/continuous-deployment.yaml

```diff
@@ -49,20 +64,35 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: gatekeeper-service
+  labels:
+    app.kubernetes.io/name: {{ include "continuous-delivery.name" . }}-gatekeeper-service
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/part-of: keptn
```

Comment:
by definition in [here](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/), `part-of` should be "The name of a higher level application this one is part of"

I think `{{ .Release.Name }}` is more sutaible for `app.kubernetes.io/instance` instead. 

However, I think that makes sense if we want to suffix it with `-{{ .Release.Namespace }}` as you suggested. So it would become
```
app.kubernetes.io/part-of: keptn-{{ .Release.Namespace }}
```
wdyt?

Reply:
Sounds good! This allows us to install Keptn in multiple namespaces and have a unique `part-of`.